python-scripts/update_hearings_with_caseReferenceNumber.py

```
@@ -0,0 +1,182 @@
+    try:
+        case_search_request = {
+            "tenantId": "kl",
+            "criteria": [{"filingNumber": filingNumber}],
+            "tenantId": "kl",
+            "RequestInfo": {
+                "apiId": "Rainmaker",
+                "authToken": auth_token,
+                "msgId": "1741081826692|en_IN",
+                "plainAccessRequest": {},
+            },
+        }
```

Pylint (3.3.7)

[warning] Duplicate key 'tenantId' in dictionary

(W0109)